tests/assembly/cmse.rs

```diff
@@ -10,14 +13,85 @@ pub trait Sized {}
 pub trait Copy {}
 
 // CHECK-LABEL: __acle_se_entry_point
-// CHECK: bxns
+// CHECK: entry_point:
```

Comment:
Question: should this be `CHECK-LABEL`?

Reply:
Oh good catch! I think this should actually be `CHECK-NEXT` since both labels need to point to the exact same block.

Reply:
Oh right, good point.

Reply:
Yes `CHECK-NEXT` works now, but it did insist that the `CHECK-LABEL` line included the trailing `:`